lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2121,6 +2127,18 @@ SubExprInfo ExprEmitter::visitSV(SampledOp op) {
   return info;
 }
 
+SubExprInfo ExprEmitter::visitSV(FileDescriptorOp op) {
+  switch (op.fdAttr().getValue()) {
+  case FileDescriptor::StdOut:
+    os << "32'h80000001";
+    break;
+  case FileDescriptor::StdErr:
+    os << "32'h80000002";
+    break;
+  }
```

Comment:
At first I wasn't sure whether we really should emit the string `32'h80000002` here, or use the existing literal emission. But I realized that the standard explicitly mentions verbatim that "32'h80000002" is the thing to do. So I think that makes total sense :+1: